Mathlib/Topology/ContinuousFunction/Algebra.lean

```diff
@@ -480,58 +480,62 @@ namespace ContinuousMap
 
 instance {α : Type*} {β : Type*} [TopologicalSpace α] [TopologicalSpace β]
     [NonUnitalNonAssocSemiring β] [TopologicalSemiring β] : NonUnitalNonAssocSemiring C(α, β) :=
-  coe_injective.nonUnitalNonAssocSemiring _ coe_zero coe_add coe_mul coe_nsmul
+  coe_injective.nonUnitalNonAssocSemiring _ coe_zero coe_add coe_mul (by intros; rfl)
```

Comment:
Do the original proofs here work if you tag `ContinuousMap.coe_pow` with `@[to_additive (reorder := 7 8)]`?

Reply:
Yep, thanks for figuring out the numbers!